client/service-container-chain/src/spawner.rs

```diff
@@ -115,9 +115,10 @@ pub struct ContainerChainSpawnParams {
 #[derive(Clone)]
 pub struct CollationParams {
     pub collator_key: CollatorPair,
-    pub orchestrator_tx_pool: Arc<FullPool<OpaqueBlock, ParachainClient>>,
-    pub orchestrator_client: Arc<ParachainClient>,
+    pub orchestrator_tx_pool: Option<Arc<FullPool<OpaqueBlock, ParachainClient>>>,
+    pub orchestrator_client: Option<Arc<ParachainClient>>,
     pub orchestrator_para_id: ParaId,
+    /// If this is `false`, then `orchestrator_tx_pool` and `orchestrator_client` must be `Some`.
```

Comment:
Could we make this receive `BuyCoreParams` instead? like you did a bit before

Reply:
I tried and it resulted in too many changes for this PR. So if you don't mind we can do it when adding the buy_core functionality for starlight.

Reply:
roadmap